CHANGELOG.md

```diff
@@ -1,10 +1,29 @@
 Changelog
 =========
 
+## 6.1.3 (2020-08-17)
+
+### Bug fixes
+
+* Persist user when set on client
+  [#770](https://github.com/bugsnag/bugsnag-cocoa/pull/770)
+
 ## 6.1.2 (2020-07-21)
 
 ### Bug fixes
 
+* Record thread information for unhandled JS errors
```

Comment:
This PR does affect vanilla Cocoa - the sharing of the crash context is probably worth highlighting as it's not obvious from the PR name.

Reply:
Reworded description